src/filter/filter-item.tsx

```diff
@@ -27,15 +30,20 @@ export const FilterItem = ({
     // =============================================================================
     const { mode } = useContext(FilterContext);
     const isMobile = mode === "mobile";
-    const [expanded, setExpanded] = useState(
-        isMobile ? true : !desktopCollapsible
+    const [collapsed, setCollapsed] = useState(
```

Comment:
can `collapsed` and `minimised` be combined to a single value?

Reply:
do you mean something like `"collapsed" | "expanded" | "maximised"`? then the check for e.g. contentHeight would be `display !== "maximised"`